apps/backend/src/main/java/no/nav/data/etterlevelse/etterlevelsemetadata/domain/EtterlevelseMetadataRepo.java

```diff
@@ -30,4 +32,9 @@ public interface EtterlevelseMetadataRepo extends JpaRepository<GenericStorage<E
 
     @Query(value = "select * from generic_storage where type = 'EtterlevelseMetadata' and data ->> 'etterlevelseDokumentasjonId' = ?1 and data -> 'kravNummer' = to_jsonb(?2)", nativeQuery = true)
     List<GenericStorage<EtterlevelseMetadata>> findByEtterlevelseDokumentasjonAndKravNummer(String behandlingId, int nummer);
+
+    @Modifying
+    @Transactional
```

Comment:
Denne annoteringen må inneholde informasjon om at metoden kan bruke omsluttende transaksjon. 

`@Transactional(propagation = PROPAGATION_REQUIRED)`

Hvis jeg husker riktig, så er dette default oppførsel med mindre vi konfigurerer TransactionManager til å oppføre seg annerledes. Men uansett greit å spesifisere for dokumentasjonens skyld (og også i tilfelle vi finner på konfigurere TM en eller annen gang som følge av en supergenial idé)

Reply:
sure oppdaterer